### ISSUE TITLE
Create an alternative for Promote your project for non-NL funding

### ISSUE THREAD

**Issue by LynseyJReynolds** [biglotteryfund/blf-alpha#3382]
For non-NL funding, the regular content in the Promote your grant section (about thanking NL players etc) isn't relevant.

We need to
- Create an alternative page that can include copy and the ability to download alternative logos
- Provide this link to the Salesforce Admins so that it can be linked to from certain offer letters
- Find a suitable home for the page in the site architecture so that it doesn't get confused with the other Promote your project page.
- Provide link to Scott so that he can make changes to the draft pages for new funding programmes

**Comment by LynseyJReynolds** [comment]
![POST AWARD Grant acknowledgement page v8 1](https://user-images.githubusercontent.com/14089243/82310003-b8685200-99bb-11ea-8eea-f527b84f222a.png)
Excuse the alignment - I only gave Colin a flat screengrab to work with!

**Comment by LynseyJReynolds** [comment]
I've created a pending page for this - https://cms.blf.digital/admin/entries/funding/142750-promoting-your-government-funded-project?site=en 

All the content (including page title) will change. 

If someone could pick this up now and add the logo download element, that'd be great. At the moment we just need the one block but we can't rule out the possibility of Gov wanting multiple colour versions in the same way as we do with our logos.

I've also talked to Scott and Rob about how integrated we want this page to be in the architecture. The initial decision has been made not to integrate it into the MYG structure and just use it as a standalone page that we can link to from other pages and the offer letter for now.
